ios/brave-ios/Sources/AIChat/Components/Settings/AIChatMenuView.swift

```diff
@@ -0,0 +1,268 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveCore
+
+private struct AIChatMenuHeaderView: View {
+  var icon: String
+  var title: String
+  
+  var body: some View {
+    HStack {
+      Image(braveSystemName: icon)
+        .foregroundStyle(Color(braveSystemName: .iconDefault))
+        .padding(.leading, 16.0)
+        .padding(.trailing, 8.0)
+        .padding(.vertical, 8.0)
```

Comment:
nit: HStack should get this `trailing` padding as regular spacing, vertical/leading padding probably should go on HStack instead